sdks/dart/client-dart-sdk/README.md

```diff
@@ -115,3 +115,58 @@ enum Readyness {
 }
 
 ```
+
+## Mobile API
+
+FeatureHub also has a mobile friendly API, that is intended to be used like a GET request and disconnect. The
```

Comment:
If you intend to use FeatureHub with Flutter for Mobile, we have an SDK that is based on REST API. The reason for this is that Mobile devices connection doesn't always stay on, so doing a GET request to get your features would be the right choice here.